NuRadioMC/simulation/simulation2.py

```diff
@@ -679,6 +685,10 @@ def _add_empty_electric_field(self, channel_id):
         self._sim_station.add_electric_field(electric_field)
 
     def _write_ouput_file(self):
+        folder = os.path.dirname(self._outputfilename)
+        if(not os.path.exists(folder)):
+            logger.warning(f"output folder {folder} does not exist, creating folder...")
```

Comment:
This is the new p3 for format, I guess?

Reply:
yes this is a fancy new p3 string formatting feature